### DISCUSSION TITLE
Support for ANSI strings

### DISCUSSION THREAD

**Discussion by ckipp01** [oyvindberg/tui-scala#46] [Ideas]
I had a use case I was playing around with today where I had a string with ANSI color codes already in the string, which obviously doesn't work. I saw [this conversation](https://github.com/fdehau/tui-rs/issues/304) in tui-rs that mentioned a couple different things, most noticeable [this ansi to `Text` converter](https://github.com/uttarayan21/ansi-to-tui). I may play around with porting this from rust -> scala because I think it'd be pretty nice to support this.

**Comment by oyvindberg** [comment]
That seems useful enough, I'll gladly accept it if you make it to a PR :)
